### DISCUSSION TITLE
vuepress markdown单独使用 作为一个组件

### DISCUSSION THREAD

**Discussion by fmkj-hhh** [vuepress/core#932] [Q&A] [answered]
怎么使用vuepress的解析markdown的功能 实现 动态的博客呢，怎么单独使用vuepress的markdown解析能力呢，有了这个能力 就可以做动态的博客网站了而不是静态的

**Comment by Mister-Hope** [comment, marked as answer]
你可以自行研究 @vuepress/markdown 并把它移植到客户端。

**Reply by fmkj-hhh** [reply]
兄弟有源码吗 怎么查看远吗，@vuepress/markdown 这个是不是支持所有的vuepress 语法啊

**Reply by Mister-Hope** [reply]
诚心的哈，你要是连源码都不知道在那，那我觉得你可能也没有能力研究并移植。所以我建议你放弃。

**Reply by fmkj-hhh** [reply]
我是java后台程序员，会一点vue，拿来用还行，也只能自己封装组件，看别人的源码，没这个本事，java后台的东西还行

**Reply by fmkj-hhh** [reply]
但是我觉得这并不妨碍我 觉得vuepress的markdown 做的很好 有开源的v-md-editor  https://code-farmer-i.github.io/vue-markdown-editor/zh/
但是我觉得这个没有vuepress 的好，所以我想要vuepress的markdown能力 完全兼容vuepress语法
